diff -uNr chwala-orig/chwala.conf chwala/chwala.conf
--- chwala-orig/chwala.conf	1970-01-01 01:00:00.000000000 +0100
+++ chwala/chwala.conf	2013-11-13 20:52:20.000000000 +0100
@@ -0,0 +1,14 @@
+Alias /chwala   /usr/share/chwala/public_html
+
+<Directory "/usr/share/chwala/public_html/">
+    AllowOverride All
+    <IfModule mod_authz_core.c>
+        # Apache 2.4
+        Require all granted
+    </IfModule>
+    <IfModule !mod_authz_core.c>
+        # Apache 2.2
+        Order Allow,Deny
+        Allow from All
+    </IfModule>
+</Directory>
